rust-symcrypt/src/lib.rs

```diff
@@ -17,6 +17,7 @@ fn symcrypt_init() {
     static INIT: Once = Once::new();
     unsafe {
         // SAFETY: FFI calls, blocking from being run again.
+        // Under the covers, the default static lib implementation exposes SymCryptModuleInit, but calls SymCryptInit for static linking
```

Comment:
What are you thoughts on this approach? it provided a safer `symcrypt-sys` API on our end since we don't expose `SymCryptInit` to `symcrypt-sys`. We could use `[cfg]` to handle this, but it that adds some complexity to the management of features and IMO will cause issues maintaining long-term.

Reply:
See more on the comments i've put on `static_WindowsDefault.c`